### DISCUSSION TITLE
Highlight / Filter disabled users

### DISCUSSION THREAD

**Discussion by nwinkelstraeter** [Mailu/Mailu#3166] [Feature requests / Ideas]
Hello everyone,

It looks like it is intended to highlight disabled users in some way as the corresponding rows get the `warning` class. https://github.com/Mailu/Mailu/blob/9d0325fa1501c9e9febbbc7eaa5093cc6f8682c7/core/admin/mailu/ui/templates/user/list.html#L31

This was added when AdminLTE 2 was still in use and now this does not seem to do anything. At least on my mailu install and on the demo instance this does not work, all lines look the same.

I guess that class is not available in AdminLTE 3 but I am not familiar enough with to suggest what else should be use instead.

Regards
Nico

**Comment by nextgens** [comment]
@ghostwheel42 is that something you can deal with in #3166 by any chance?

**Reply by DjVinnii** [reply]
Do we still want this? I could open a quick PR to add the table-warning class:
![image](https://github.com/user-attachments/assets/d238e8e3-b109-4748-98d3-643f41dc11b5)

**Reply by nextgens** [reply]
sure; please ensure we don't use that deprecated class anywhere else either if you do
